charm/docs/tutorial/getting-started.md

```diff
@@ -1,31 +1,34 @@
+<!-- This tutorial should be updated in the src-docs directory. -->
+
 # Getting Started
 
 ## What you’ll do
+
 - Deploy the NetBox charm.
 - Integrate with Redis using the redis-k8s charm.
 - Integrate with the PostgreSQL K8s charm.
 - Integrate with S3 for storage.
 - Expose the NetBox charm with Traefik k8s.
 - Create a super user.
 
-Through the process, you'll verify the workload state, and log in to
+Through the process, you’ll verify the workload state, and log in to
 your NetBox instance.
 
 ## Requirements
+
 - Juju 3 installed.
 - Juju controller that can create a model of type kubernetes.
```

Comment:
I would change these lines to something like 

` - Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by using a Multipass VM as outlined in this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)`

Perhaps we don't need to specify MicroK8s. It doesn't seem like the user will need to visit a website, so hopefully the rest of the tutorial remains the same if the user decides to use Multipass.